### DISCUSSION TITLE
Tags

### DISCUSSION THREAD

**Discussion by iamaravindh** [outline/outline#2256] [Ideas]
Currently outline not supporting tags, its impossible to search based on folder when we have multiple dependency with same folder, if tags are added it easier to sort and filter

**Comment by tommoor** [comment]
Hi – there is an issue for tags here: https://github.com/outline/outline/issues/765

I think at this time we're avoiding building this in favor of improving search more generally, the less work we can require of folks to organize documents – the easier the product is to use. The current hierarchy system already requires a lot of folks. If you have some good examples of where tags would have been useful to improve search results those would be great to see!

**Reply by sudoexec** [reply]
It's useful in filtering a topic or some similar content.
The search system is for finding accurate words. But sometimes maybe you don't know what word to search, or you want to read all content about a tag in the knowledgw base,so you want to do fuzzy search using tags.
It's just like tags in the blog system.

**Comment by BRogers80** [comment]
Dear team,

we use outline for multiple use cases and have 5 different collections.
With a tag system we can make sure that content is tagged with appropriate words. We have our own bridge from our ticketing system to outline with the api. A tag system will allow us to start a search across given fixed words. (Give me all installation guidelines for customer xy independently if these documents are under category a or b). With a tag system I can create documents from my ticket system with the api and I am absolutely sure that all content is found. Sometime a article is relevant for a search but is in a different collection. (Our employee want to search the installation guideline for the product a for customer b, but a common installation guideline for the product also exists in another collection). So filtering on collections does not always make sense for us. With a tag system I can make sure if the installation guideline for customer b doe not exist the generell guideline is displayed.
Sure, I can search in outline and in different places. But with a api search I can make sure to get all relevant tagged documents.

regards

Flo

**Comment by dsmrs** [comment]
Tags are not for search, but to filter. It can also be used in other features, like a Kanban view, see: https://github.com/outline/outline/discussions/3440

**Reply by danfoxley** [reply]
Yes, for example, if limited to a library of tags, either forced by application functionality or by team consensus. 🤦 Can it be implemented with an integration?

**Comment by retiarylime** [comment]
I am upvoting this idea. Tags are important to sort and categorise the docs.

**Comment by atandy** [comment]
Tags would still be nice. It seems that this is still an open issue. My use case is that I have documents that I am creating that need to have videos added to them at some point. I'd like to be able to add a tag or todo tag to the doc to refer back to/filter all the docs that require videos being added to them. 

The alternative is to keep a separate list somewhere else that includes urls for all of the documents that still have this requirement on them.

**Comment by juliocoelho** [comment]
The main advantage of a tag system is that it enables the creation of smart collections. These collections can automatically group documents based on the inclusion or exclusion of specific tags.

For example, you could create separate collections for administrative processes, HR & Benefits, and development processes, with each collection containing relevant documents. Some of these documents can be tagged with "onboarding," so they could be included in an "onboarding" smart collection, regardless of their original collection.

Additionally, you could use a tag like "draft" to refine your onboarding collection. For instance, the "onboarding" smart collection could include all documents tagged with "onboarding" but exclude those tagged as "draft."

**Comment by McFlyPartages** [comment]
Hello everyone,

I see that this topic has been going on since 2021.

For my part, this is one of the important things that outline lacks in centralized documentation management.

I think it would be really appreciated, there are few documentation applications or publications without a tag.

Best regards

Anthony, McFly

**Comment by baditaflorin** [comment]
I would also want to see tags or some sort of #tags #system where when i write a hashtag and click on it would take me to a list of all documents that are using that tag or hashtag

**Comment by swinis** [comment]
+1

**Comment by Brukkil** [comment]
+1

**Comment by pavelcdn** [comment]
+1

**Comment by DavidDieguezCodex** [comment]
I was reviewing Outline for my company and I found this after not being able to create tags. I think this is going to be a pass for me sorry....

**Comment by NeptunedGmbH** [comment]
I really like outline but I do miss Tags. E.g. for process documentation tags would be really helpful. I'd like to tag all involved departments in a process. The other option I'd really like to see would be Workflowy-Style Mirrors. In Outline I'd only want to mirror a whole document since thats the smallest bit of information but that would still be great. That way I could just add a Collection or Paren Document for each Department and then have all the corresponding processes in their.

**Comment by CanalWood** [comment]
+1

Hello,
My team use Outline and today I would like to find a way to have a filter on related document. 
TAG implementation will be the solution. 

Do you have any info related to this implementation?
Thx

**Comment by hdinguyen** [comment]
+1

**Comment by defcon1702** [comment]
+1 for tags. Also for a better "worktogether" with Obsidian and systems like Para or Zettlkasten

**Comment by defcon1702** [comment]
Maybe you want to take a look? :)
https://github.com/outline/outline/discussions/11603

**Comment by MystikReasons** [comment]
+1

**Comment by julbdx** [comment]
+1

**Comment by thomaslc08** [comment]
+1

**Comment by pavelcdn** [comment]
Hello! Is the tags functionality currently on the product roadmap for 2026? Many users are looking forward to this feature for better categorization. Would appreciate any insight on the estimated timeline

**Reply by defcon1702** [reply]
Two PRs for tags without further responds so hard to tell.

**Comment by jammin88** [comment]
+1

**Comment by thaibinhindex** [comment]
+1

**Comment by JanJK2011** [comment]
+1

**Comment by kameleon1er** [comment]
+1 could be useful to align process with "Obsidian-Properties-tags" 💯
